release/edgex-ui-go.yaml

```diff
@@ -1,11 +1,11 @@
 ---
 name: 'edgex-ui-go'
-version: '3.1.0'
-releaseName: 'napa'
+version: '4.0.0'
+releaseName: 'odessa'
 releaseStream: 'main'
 lts: true
```

Comment:
@cloudxxx8 @cherrycl Since edgex-ui-go is not in our LTS scope, do we need to change this value to `false`?

Reply:
since we have set the odessa as the LTS version in the Jenkins job, we can still set it to true
this setting is for build job only